### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by DarkOsa** [kangyu-california/PersistentWindows#160] [General]
Been dealing for months with this issue until I pretty much randomly stumbled upon someone mentioning this tool somewhere on Reddit. Thank you, monitors exiting from sleep exactly like they should now, with their corresponding windows kept in place!

**Comment by samsquamsch** [comment]
In lieu of making another thanks thread, I'm bumping this one. Thank you so much! I had become dulled to the tedium of rearranging my applications every single time I woke my computer up and found everything bunched together on my main monitor. No more!!! I wish I found this program years ago.

**Comment by masenf** [comment]
Sharing my **appreciation and gratitude** for the maintainer taking the time to maintain an essential, "invisible" application (and to the original author for creating it).

I had been enjoying PersistentWindows so much for years, that I kind of forgot it was there, because everything just worked...until github changed their formatting last week, breaking the auto-update, and ultimately causing the app to crash and be useless.

In these moments, I was reminded of the horrid UX of multi-monitor windows just rearranging themselves however they please, so I went looking for the source of the issue and found a fix in 5.41 (and again my sanity!)

It's the little things, like my stuff being where I left it, that keep "my cup full" to better brave the following death by 1000 cuts that is every day working as a developer on Windows 😏; for that, your work is greatly enjoyed.

**Comment by j-autuick** [comment]
Also, want to bump this, and say thank you. I have a multi-computer, multiple display KVM, and the windows moving whenever I switched systems was driving me BATTY! I was dissuaded from even using my windows box, but now I might give it some more burn. I can't believe Microsoft doesn't handle this better. Seriously, I can use my Linux and Mac systems without issue, but Microsoft, no. I guess what can you expect from a poor, small startup, Microsoft just doesn't have the resources to spend.

**Comment by CalculonPrime** [comment]
I was pleased to find this software works to restore windows on a second monitor that Windows inexplicably moves away when I switch the primary monitor connection with a KVM.  Why can't we just switch off whatever in Windows is doing this?  I heard that there was a "Windows preview" release that had this feature, but the feature itself never got pushed to an actual released version of Windows.

Reminds of a funny sketch on the Simpsons where this smart guy built a device that attaches to a toilet in Australia and reverses the flush direction to make flush counterclockwise like in the United States.  (I suspect the water jet positioning dominates the Coriolis Force at such small scales, but still it was funny, and I was reminded of that sketch having to install this software to undo Windows behavior.)

**Comment by arkdamenen** [comment]
Previously I thought that I can live with DP problem, but one day I become tired of it.
Googled a bit and got a link I think from LTT forum to this wonderful fork.

Thank you so much for your great work!
